### ISSUE TITLE
false positive, unused variable, CHECK

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1225]
```abap
DATA bar TYPE abap_bool.
CHECK bar = abap_false.
```
